### PR TITLE
Implement Get SCAT Analysis page

### PR DESCRIPTION
**Pull request by nikiburggraf**
Includes placeholder images representing falsified real data with corresponding dummy IDs, returned by the dummy API client

Also includes testing for said dummy API client

### CODE REVIEW

File: src/genetic_forensic_portal/app/client/gf_api_client.py [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+MISSING_UUID_ERROR = "uuid is required"
+
 SAMPLE_UUID = "this-is-a-uuid"
 NO_METADATA_UUID = "this-is-a-differentuuid"
+NOT_FOUND_UUID = "not-found-uuid"
+NOT_AUTHORIZED_UUID = "not-authorized-uuid"
+
+UUID_LIST = [SAMPLE_UUID, NO_METADATA_UUID, NOT_FOUND_UUID, NOT_AUTHORIZED_UUID]
+
+SAMPLE_IMAGE_PATH = "src/genetic_forensic_portal/resources/sample_images/"
```
Comment (lsetiawan) [inline src/genetic_forensic_portal/app/client/gf_api_client.py:13]: Let's not harcode this, but rather use the `pathlib` library to build this path relatively to this file

**Review by lsetiawan — CHANGES_REQUESTED** [review]
One small change to avoid hardcoding of path since this might change b/w operating systems

File: src/genetic_forensic_portal/app/client/gf_api_client.py (hunk 1 above)
Comment (nikiburggraf) [reply src/genetic_forensic_portal/app/client/gf_api_client.py]: Should be resolved in new revision

**Review by lsetiawan — APPROVED** [review]
LGTM

**Pull request merged**